src/rules/javascript/shallow_wrapper.js

```diff
@@ -0,0 +1,35 @@
+import * as messages from 'messages/javascript';
+
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      if (node.callee.name === 'XPCNativeWrapper' &&
+          node.callee.type === 'Identifier' &&
+          // If only one argument is used it's okay;
```

Comment:
Moving this to the top of the rule with supporting links would save on the duplication - it's not so spread out as to be an issue.